src/client/public-openfin/launcher.json

```diff
@@ -72,7 +72,6 @@
         "defaultHeight": 56,
         "defaultTop": 160,
         "defaultLeft": 30,
-        "backgroundColor": "#313131",
```

Comment:
I thought the `backgroundColor` was supposed to help, per OF docs extract ..
(assuming it is set to dark theme ofc) .. so how come we still get 2s or so of white flash?